example/ipsec_api/README

```diff
@@ -0,0 +1,154 @@
+Copyright (c) 2014-2018, Linaro Limited
+Copyright (c) 2020, Nokia
+All rights reserved.
+
+SPDX-License-Identifier:        BSD-3-Clause
+
+1. Intro
+
+The IPsec API example application "odp_ipsec_api" functions as a simple L3 IPv4
+router which supports IPsec 3DES cipher and HMAC-MD5 authentication in both the
```

Comment:
Yes, 3DES and HMAC-MD5 are supported but not at the same time as one might expect. It might be useful to state what is supported:
- ESP with 3DES or NULL encryption algorithm, with NULL authentication
- AH with HMAC-MD5 or HMAC-SHA-256 integrity algorithm

I get the feeling that the original author(s) may have confused AH and the authentication algorithm of ESP, resulting in a bit weird examples that attempt to support combined ESP+AH but not ESP with both confidentiality and integrity.

Reply:
Okay, will add this is V2.